core/prometheus_exporter_test.go

```diff
@@ -0,0 +1,11 @@
+package core
+
+import "testing"
+
+func TestCollector(t *testing.T) {
+	// TODO
```

Comment:
When trying to initialize a new context and register services the `ipvs.AddService` function returns an error: https://github.com/kobolog/gorb/blob/master/core/context.go#L195

Most probably is because I'm not doing the proper test set up procedure as it is done in [here](https://github.com/kobolog/gorb/blob/master/main.go#L81)

I haven't look to deep into the error. I can open another PR trying to add a more complete test coverage.